packages/sdk/CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# How to Contribute to the SDK
+
+## Local Environment Setup
+
+### Global Dependencies
+
+Clone the repo (ideally your own fork of it) and initialize the global
+dependencies like Node.js, NPM, etc. We use [`asdf`](https://asdf-vm.com/) to
+manage these, so assuming we're located in this directory (i.e. `packages/sdk/`)
+we can run the following command to install them:
+
+```shell
+asdf install
+```
+
+If you prefer to use another tool make sure you use the same versions that are
+specified in the `.tool-versions` files [in this current directory, and
+recursively going up until you reach the root of the
+repo](https://asdf-vm.com/guide/getting-started.html#_6-set-a-version).
+
+### Local Dependencies
+
+You can install the package's dependencies by using `pnpm` or `npm`:
+
+```shell
+pnpm install
```

Comment:
If we have people install root deps, should we do `npx pnpm install`?

Reply:
`asdf install` should install `pnpm` as a standalone executable I think.
We should probably revisit the main `CONTRIBUTING.md` docs since `pnpm` is used across the repo.

Reply:
nice yes, cool since that's in `.tool-versions` that works